russh/src/parsing.rs

```diff
@@ -107,6 +111,29 @@ pub struct TcpChannelInfo {
     pub originator_port: u32,
 }
 
+#[derive(Debug)]
+pub struct StreamLocalChannelInfo {
+    pub client_socket_path: String,
+    pub server_socket_path: String,
+}
+
+impl StreamLocalChannelInfo {
+    fn new(r: &mut Position) -> Result<Self, crate::Error> {
+        // NEED HELP: is this completely correct?
+        let client_socket_path = std::str::from_utf8(r.read_string().map_err(crate::Error::from)?)
+            .map_err(crate::Error::from)?
+            .to_owned();
+        let server_socket_path = std::str::from_utf8(r.read_string().map_err(crate::Error::from)?)
```

Comment:
Same here

Reply:
Then how to determine server_socket_path? Or should it not even be known? Once again, confused since tcp/ip has both the client's and the server's, but apparently streamlocal only has one?